bob-common/src/mapper.rs

```diff
@@ -291,23 +292,26 @@ impl Virtual {
             .collect()
     }
 
-    pub fn get_operation(&self, key: BobKey) -> (VDiskId, Option<DiskPath>) {
+    pub fn get_operation(&self, key: BobKey) -> (VDiskId, Option<SmallVec<[DiskPath; 1]>>) {
         let virt_disk = self.get_vdisk_for_key(key).expect("vdisk not found");
-        let disk = virt_disk.replicas().iter().find_map(|disk| {
-            if disk.node_name() == &self.local_node_name {
-                Some(DiskPath::from(disk))
-            } else {
-                None
+        let mut disks = None;
+        for replica in virt_disk.replicas() {
+            if replica.node_name() == &self.local_node_name {
+                if disks.is_none() {
+                    disks = Some(SmallVec::<[DiskPath; 1]>::new());
+                }
+                disks.as_mut().unwrap().push(DiskPath::from(replica));
```

Comment:
Use `expect()` instead of unwrap, because it is expected, that `disks` is not `None` here